sdk/appconfiguration/app-configuration/test/internal/node/package.spec.ts

```diff
@@ -29,7 +24,7 @@ describe("packagejson related tests", () => {
     } catch (e) {
       // For unit tests
       packageJsonContents = JSON.parse(
-        fs.readFileSync(path.join(__dirname, "../package.json"), { encoding: "utf-8" })
+        fs.readFileSync(path.join(__dirname, "../../package.json"), { encoding: "utf-8" })
```

Comment:
I'm confused about why this path changed.

Reply:
looks like the file has been moved to the node folder.

Reply:
yes I removed the isNode check and move this test file to the node folder.

Reply:
Looks that it's failing in integration tests now